metadata.rb

```diff
@@ -8,11 +8,10 @@
 source_url       'https://github.com/dnsimple/chef-pdns'
 issues_url       'https://github.com/dnsimple/chef-pdns/issues'
 
-chef_version '>= 12.5'
+chef_version '>= 12.14'
 
 supports 'ubuntu', '>= 14.04'
 supports 'debian', '>= 8.0'
 supports 'centos', '>= 6.0'
 
 depends 'apt'
-depends 'yum'
```

Comment:
That was an error in documentation and I have corrected it. As of Chef 12.14 the yum_repository resources are now a core resource inside of Chef.